pkg/minikube/bootstrapper/exec_runner.go

```diff
@@ -34,10 +35,31 @@ import (
 // It implements the CommandRunner interface.
 type ExecRunner struct{}
 
+// splitCommandString splits a command string like `ls -l` into the command `ls` and `-l`
+func splitCommandString(cmd string) (string, []string) {
```

Comment:
I'm not yet convinced that this will work without a lot of extra hacks, but https://godoc.org/github.com/google/shlex will work better than implementing our own splitter, as it properly handles quotes.

Reply:
I agree. `shlex` would be better. I found about that after committing the changes in this PR.

I am going ahead and implementing the same using shlex